src/pages/404.tsx

```diff
@@ -1,17 +1,26 @@
 import * as React from "react";
 import { Header, Icon, Grid } from "semantic-ui-react";
+import { default as DefaultLayout } from "../components/Layout";
```

Comment:
`import DefaultLayout from "../components/Layout";` should works here

Reply:
Yes, it looks better - fixed